cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/securityContext.xml

```diff
@@ -21,24 +21,35 @@
 -->
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xmlns:p="http://www.springframework.org/schema/p"
-       xmlns:sec="http://www.springframework.org/schema/security"
+       xmlns:mvc="http://www.springframework.org/schema/mvc"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
-       http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd">
+       http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc.xsd">
+
+    <description>
+        Security configuration for sensitive areas of CAS : status and statistics.
+    </description>
+
+    <bean id="config" class="org.pac4j.core.config.Config">

```

Comment:
I updated the pull request. Thanks to new constructors, the config definition is a little shorter.

Notice that a 401 HTTP error is returned instead of a 403 HTTP error for Spring Security. To generate a 403 HTTP error with pac4j, you need this configuration:

```
    <bean id="config" class="org.pac4j.core.config.Config">
        <constructor-arg name="client">
            <bean class="org.pac4j.http.client.direct.AnonymousClient" />
        </constructor-arg>
        <constructor-arg name="authorizer">
            <bean class="org.pac4j.http.authorization.IpRegexpAuthorizer">
                <constructor-arg name="regexpPattern" value="${cas.securityContext.adminpages.ip:127\.0\.\.1}" />
            </bean>
        </constructor-arg>
    </bean>

    <mvc:interceptors>
        <mvc:interceptor>
            <mvc:mapping path="/status/**" />
            <mvc:mapping path="/statistics/**" />
            <bean class="org.pac4j.springframework.web.RequiresAuthenticationInterceptor">
                <constructor-arg name="config" ref="config" />
                <constructor-arg name="clientName" value="AnonymousClient" />
                <constructor-arg name="authorizerName" value="IpRegexpAuthorizer" />
            </bean>
        </mvc:interceptor>
    </mvc:interceptors>
```